priv/leveled_multi.schema

```diff
@@ -39,8 +39,8 @@
 %% Can be lz4 or native (which will use the Erlang native zlib compression)
 %% within term_to_binary
 {mapping, "multi_backend.$name.leveled.compression_method", "riak_kv.multi_backend", [
-  {default, lz4},
-  {datatype, atom},
+  {datatype, {enum, [native, lz4, none]}},
+  {default, native}
```

Comment:
Sure to change the default here?

Reply:
It was a mistake.  The default for leveled backend was changed to native, but in that fix, forgot to change the multi-backend.  So this realigns the two again.

For riak, as it supports a HTTP API, there tends to be a lot of stuff that ends up being stored after base64 encoding.  In this case we get a 30% compression improvement native via lz4. The throughput improvement we've seen with lz4 due to efficient CPU has always been no more than about 3% except for contrived situations.